### PR TITLE
[zh]: Rsyc content/zh/docs/setup/production-environment/tools/kubeadm setup-2

### PR DESCRIPTION
**Pull request by zaunist**
Part of https://github.com/kubernetes/website/issues/33475   setup-2

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -899,35 +927,129 @@ options.
...
+
+<!--
+Example:
+* kubeadm is at {{< skew latestVersion >}}
+* `kubernetesVersion` must be at {{< skew latestVersion >}} or {{< skew prevMinorVersion >}}
+-->
+
+例子：
+* kubeadm 的版本为 {{< skew latestVersion >}}。
+* `kubernetesVersion` 必须在 {{< skew latestVersion >}} 或者 {{< skew prevMinorVersion >}}。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:970]:
```suggestion
* `kubernetesVersion` 必须为 {{< skew latestVersion >}} 或者 {{< skew prevMinorVersion >}}。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -899,35 +927,129 @@ options.
...
+
+<!--
+### kubeadm's skew against the kubelet
+-->
+
+### kubeadm 对 kubelet 版本的倾斜度
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:976]:
```suggestion
### kubeadm 中 kubelet 的版本偏差
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 3]
```diff
@@ -899,35 +927,129 @@ options.
...
+
+<!--
+Similarly to the Kubernetes version, kubeadm can be used with a kubelet version that is the same
+version as kubeadm or one version older.
+-->
+
+与 Kubernetes 版本类似，kubeadm 可以与 kubelet 版本一起使用，
+该 kubelet 版本与 kubeadm 的版本相同，或者比它大一个版本。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:984]:
```suggestion
与 Kubernetes 版本类似，kubeadm 可以使用与 kubeadm 相同版本的 kubelet，
或者比 kubeadm 老一个版本的 kubelet。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 4]
```diff
@@ -899,35 +927,129 @@ options.
...
+
+例子：
+* kubeadm 的版本为 {{< skew latestVersion >}}
+* 主机上的 kubelet 版本必须为 {{< skew latestVersion >}} 或者 {{< skew prevMinorVersion >}}
+
+<!--
+### kubeadm's skew against kubeadm
+-->
+
+### kubeadm 对 kubeadm 的倾斜度
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:1000]:
```suggestion
### kubeadm 支持的 kubeadm 的版本偏差
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 5]
```diff
@@ -899,35 +927,129 @@ options.
...
 <!--
-These resources provide more information on supported version skew between kubelets and the control plane, and other Kubernetes components:
+Example for `kubeadm join`:
+* kubeadm version {{< skew latestVersion >}} was used to create a cluster with `kubeadm init`
+* Joining nodes must use a kubeadm binary that is at version {{< skew latestVersion >}}
 -->
-这些资源提供了有关 kubelet 与控制平面以及其他 Kubernetes 组件之间受支持的版本倾斜的更多信息：
+
+`kubeadm join` 的例子：
+* 版本为 {{< skew latestVersion >}} 的 kubeadm 被用来使用 `kubeadm init` 创建一个集群。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:1027]:
```suggestion
* 使用 `kubeadm init` 创建集群时使用版本为 {{< skew latestVersion >}} 的 kubeadm。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 6]
```diff
@@ -899,35 +927,129 @@ options.
...
 <!--
-* Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
-* Kubeadm-specific [installation guide](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl)
+Nodes that are being upgraded must use a version of kubeadm that is the same MINOR
+version or one MINOR version newer than the version of kubeadm used for managing the
+node.
 -->
-* Kubernetes [版本和版本偏斜政策](/zh/docs/setup/release/version-skew-policy/)
-* Kubeadm-specific [安装指南](/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl)
+正在升级的节点必须使用与管理节点所用的 kubeadm 版本相同的 MINOR 版本或比其新的一个 MINOR 版本的 kubeadm。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:1035]:
```suggestion
对于正在升级的节点，所使用的的 kubeadm 必须与管理该节点的 kubeadm 具有相同的
MINOR 版本或比后者新一个 MINOR 版本。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 7]
```diff
@@ -899,35 +927,129 @@ options.
...
+-->
+`kubeadm upgrade`的例子:
+* 用于创建或升级节点的 kubeadm 版本为 {{< skew prevMinorVersion >}}。
+* 用于升级节点的 kubeadm 版本必须为 {{< skew prevMinorVersion >}} 或 {{< skew latestVersion >}}。
+
+<!--
+To learn more about the version skew between the different Kubernetes component see
+the [Version Skew Policy](https://kubernetes.io/releases/version-skew-policy/).
+-->
+要了解更多关于不同 Kubernetes 组件之间的版本倾斜，请参见
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:1051]:
```suggestion
要了解更多关于不同 Kubernetes 组件之间的版本偏差，请参见
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 8]
```diff
@@ -899,35 +927,129 @@ options.
...
+[版本倾斜政策](https://kubernetes.io/releases/version-skew-policy/)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:1052]:
```suggestion
[版本偏差策略](https://kubernetes.io/releases/version-skew-policy/)。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 9]
```diff
@@ -225,10 +222,9 @@ If the reload and restart are successful, the normal `kubeadm init` workflow con
 -->
 ### 当使用 `kubeadm init`时的工作流程
 
-当调用 `kubeadm init` 时，kubelet 配置被编组到磁盘上的 `/var/lib/kubelet/config.yaml` 中，
-并且上传到集群中的 ConfigMap。
-ConfigMap 名为 `kubelet-config-1.X`，其中 `X` 是你正在初始化的 kubernetes 版本的次版本。
-在集群中所有 kubelet 的基准集群范围内配置，将 kubelet 配置文件写入 `/etc/kubernetes/kubelet.conf` 中。
+当调用 `kubeadm init` 时，kubelet 的配置会被写入磁盘 `/var/lib/kubelet/config.yaml`，
+并上传到集群 `kubee-system` 命名空间的`kubelet-config` ConfigMap。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md:226]:
```suggestion
并上传到集群 `kubee-system` 命名空间的 `kubelet-config` ConfigMap。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 10]
```diff
@@ -225,10 +222,9 @@ If the reload and restart are successful, the normal `kubeadm init` workflow con
...
+一个 kubelet 配置文件也被写入 `/etc/kubernetes/kubelet.conf`，其中包含集群内所有 kubelet 的基线配置。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md:227]:
```suggestion
kubelet 配置信息也被写入 `/etc/kubernetes/kubelet.conf`，其中包含集群内所有 kubelet 的基线配置。
```

File: content/zh/docs/setup/best-practices/certificates.md [hunk 11]
```diff
@@ -285,3 +373,22 @@ These files are used as follows:
...
 
+<!--
+The following files illustrate full paths to the files listed in the previous table:
+
+```
+/etc/kubernetes/admin.conf
+/etc/kubernetes/kubelet.conf
+/etc/kubernetes/controller-manager.conf
+/etc/kubernetes/scheduler.conf
+```
```
[the remark this answers is not in the document]
Comment (zaunist) [reply content/zh/docs/setup/best-practices/certificates.md:384]: 好的，明白了

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 12]
```diff
@@ -302,6 +299,14 @@ To customize control plane components, including optional IPv6 assignment to liv
 -->
 要自定义控制平面组件，包括可选的对控制平面组件和 etcd 服务器的活动探针提供 IPv6 支持，请参阅[自定义参数](/zh/docs/setup/production-environment/tools/kubeadm/control-plane-flags/)。
 
+<!--
+To reconfigure a cluster that has already been created see
+[Reconfiguring a kubeadm cluster](/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure).
+-->
+
+要重新配置一个已经创建的集群，请参见
+[重新配置一个 kubeadm 集群](/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure)。
```
[the remark this answers is not in the document]
Comment (zaunist) [reply content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:308]: 好的

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 13]
```diff
@@ -517,43 +520,70 @@ for `kubeadm`.
...
+admission controller that restricts what labels can be self-applied by kubelets on node registration.
+The admission controller documentation covers what labels are permitted to be used with the kubelet `--node-labels` option.
+The `node-role.kubernetes.io/control-plane` label is such a restricted label and kubeadm manually applies it using
+a privileged client after a node has been created. To do that manually you can do the same by using `kubectl label`
+and ensure it is using a privileged kubeconfig such as the kubeadm managed `/etc/kubernetes/admin.conf`.
+-->
+默认情况下，kubeadm 启用 [NodeRestriction](/zh/docs/reference/access-authn-authz/admissiontrollers/#noderestriction)
+准入控制器来限制 kubelets 在节点注册时可以应用哪些标签。准入控制器文档描述 kubelet `--node-labels` 选项允许使用哪些标签。
+其中 `node-role.kubernetes.io/control-plane` 标签就是这样一个受限制的标签，
+kubeadm 在节点创建后使用特权客户端应用此标签。
```
Comment (javadoors) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:540]:
```suggestion
kubeadm 在节点创建后使用特权客户端手动应用此标签。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 14]
```diff
@@ -517,43 +520,70 @@ for `kubeadm`.
...
 
 <!--
-This will remove the `node-role.kubernetes.io/master` taint from any nodes that
-have it, including the control-plane node, meaning that the scheduler will then be able
+This will remove the `node-role.kubernetes.io/control-plane` and
+`node-role.kubernetes.io/master` taints from any nodes that have them,
+including the control plane nodes, meaning that the scheduler will then be able
 to schedule Pods everywhere.
 -->
 这将从任何拥有 `node-role.kubernetes.io/master` taint 标记的节点中移除该标记，
```
Comment (javadoors) [inline content/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:575]:
```suggestion
这将从任何拥有 `node-role.kubernetes.io/control-plane` 和 `node-role.kubernetes.io/master` 污点标记的节点中移除该标记，
```

**Pull request merged**